### ISSUE TITLE
Add support for args

### ISSUE THREAD

**Issue by tobiasdiez** [tobiasdiez/unplugin-storybook-vue#13]
Proposal:
```vue
<script setup>
import Button from './Button.vue'

const args = defineArgs({
  color: { control: 'color' },
})
</script>
<template>
  <Stories :initialArgs="{ label: 'Button' }">
     <Story title="Primary" :initalArgs="{ color: 'red' }">
        <Button v-bind="args" />
      </Story>
  </Stories>
</template>
```

The parameter to `defineArgs` is optional and is automatically mapped to `argTypes` in the default export.

@ IanVS what do you think?

**Comment by JoJk0** [comment]
@tobiasdiez any update on this? I love using `.vue` files instead of `CSF` but I've had to ditch it as lack of args customization is a show stopper for me 👎

**Comment by tobiasdiez** [comment]
I agree for simple templates there is little advantages over having multiple story defs. But if the story template gets more involved, you don't want to copy-paste it to every variant of the args. But probably this use case can be taken care of by https://github.com/tobiasdiez/storybook-vue-addon/issues/1.
